htmlproject.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Html Project</title>
+    <link rel="stylesheet" href="htmlprojectstyles.css">
+</head>
+
+<body>
+    <header>
+        <div class="navBar">
```

Comment:
Try to stick to using kebab case for naming in HTML. Check [this guideline](http://udacity.github.io/frontend-nanodegree-styleguide/index.html) out to learn more about HTML conventions.